boards/arm/stm32f429i_disc1/stm32f429i_disc1_defconfig

```diff
@@ -23,6 +23,12 @@ CONFIG_PINMUX=y
 # enable GPIO
 CONFIG_GPIO=y
 
+# enable FMC
+CONFIG_MEMC=y
```

Comment:
Please follow [board porting recommendations](https://docs.zephyrproject.org/latest/guides/porting/board_porting.html?highlight=board%20porting#general-recommendations) and let this to application responsibility.